content/docs/applicationmobility/_index.md

```diff
@@ -6,6 +6,8 @@ Description: >
   Application Mobility
 ---
 
+{{% pageinfo color="primary" %}} The CSM Application Mobility Helm deployment will be deprecated in a future release. It is highly recommended that you use CSM Operator going forward. {{% /pageinfo %}}
```

Comment:
Please provide a link to the CSM Operator as it relates to deploying Application Mobility.  Thanks.

Reply:
My recommendation here would be to leave the documentation as is